.config/pa11yci.config.js

```diff
@@ -8,7 +8,12 @@ const filterStories = (stories) => {
   const exclude = [
     "Overview/",
     "Foundation/",
-    "Guides/",
+    "Guides/Provider",
+    "Guides/Theming",
+    "Guides/Styling",
+    "Guides/Forms",
+    "Guides/Visualizations",
+    "Guides/Layout",
```

Comment:
What do you think about moving the `HvTable` docs to `Components/`?
It feels really out of place under "Guides", as it's a component and its docs the typical component props + samples, unlike the other guides that we have here

Besides, this is also not being includes in Applitools

Reply:
Yes, makes sense! but maybe we should add the table stories to the visualisations. In the project status, this component is under visualisations, and also in DS.

Reply:
Yeah, sure! It's where we previously had it